package.json

```
@@ -3,14 +3,32 @@
   "version": "18.0.0",
   "private": true,
   "description": "Проект «Киноман» от HTML Academy",
+  "browserslist": [
+    "last 2 versions",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all",
+    "not < 0.25%"
+  ],
   "scripts": {
-    "lint": "eslint src/"
+    "lint": "eslint src/",
+    "build": "webpack --mode production",
+    "start": "webpack serve --mode development --open"
   },
   "devDependencies": {
     "eslint": "8.19.0",
     "eslint-config-htmlacademy": "6.0.0"
   },
   "engines": {
     "node": "16"
+  },
+  "dependencies": {
+    "@babel/core": "^7.18.6",
+    "@babel/preset-env": "^7.18.6",
+    "babel-loader": "^8.2.5",
+    "copy-webpack-plugin": "^10.2.4",
+    "webpack": "^5.73.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.9.3"
```

Версии нужно фиксировать жестко, без модификаторов
```suggestion
    "@babel/core": "7.18.6",
    "@babel/preset-env": "7.18.6",
    "babel-loader": "8.2.5",
    "copy-webpack-plugin": "10.2.4",
    "webpack": "5.73.0",
    "webpack-cli": "4.10.0",
    "webpack-dev-server": "4.9.3"
```